client/components/shared/NewBankForm.js

```diff
@@ -9,14 +9,18 @@ export default class NewBankForm extends React.Component {
     constructor(props) {
         has(props, 'expanded');
         super(props);
-        this.state = {
-            expanded: this.props.expanded,
-            hasCustomFields: false,
-            customFields: []
-        };
+        let bank = store.getStaticBanks()[0];
+        this.state = Object.assign(
```

Comment:
How about:

`var state = this.computeState(bank);
state.expanded = this.props.expanded;
this.state = state;`